batch2/batch/worker.py

```diff
@@ -424,7 +439,7 @@ def job_id(self):
 
     @property
     def id(self):
-        return (self.batch_id, self.job_id)
+        return self.batch_id, self.job_id
```

Comment:
I'm not sure if we're consistent about this, but I prefer parens when forming tuples.